config/crd/external/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -0,0 +1,763 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
why is this needed?

Reply:
Yes. I also don't understand why this is needed.

Reply:
because odh-model-controller look for this by default